samples/java_springboot/19.custom-dialogs/src/main/java/com/microsoft/bot/sample/customdialogs/DialogBot.java

```diff
@@ -48,6 +50,9 @@ public DialogBot(
     protected CompletableFuture<Void> onMessageActivity(
         TurnContext turnContext
     ) {
+        Logger.getLogger(DialogBot.class.getName()).info("Running dialog with Message Activity.");
```

Comment:
```suggestion
        Logger.getLogger(DialogBot.class).info("Running dialog with Message Activity.");
```

Reply:
The getLogger() needs to receive a String, it cannot receive a class

Reply:
this should be the solutions: `LoggerFactory.getLogger(DialogBot.class).info("Running dialog with Message Activity.");`

Reply:
Sorry, my mistake the change should be this:
`LoggerFactory.getLogger(DialogBot.class).info("Running dialog with Message Activity.");`